Sources/XCTest/XCAbstractTest.swift

```diff
@@ -26,6 +26,37 @@ public class XCTest {
         fatalError("Must be overridden by subclasses.")
     }
 
+    /// The `XCTestRun` subclass that will be instantiated when the test is run
+    /// to hold the test's results. Must be overridden by subclasses.
+    public var testRunClass: AnyClass {
```

Comment:
Nice catch. Updated to use `AnyClass?` here. Submitted https://bugs.swift.org/browse/SR-1164 (with radar) to track changing the type in the SDK overlay.